packages/add-nx-to-monorepo/src/add-nx-to-monorepo.ts

```diff
@@ -49,7 +49,7 @@ async function addNxToMonorepo() {
   let scriptOutputs = {} as { [script: string]: string };
   let useCloud: boolean;
 
-  if (parsedArgs.yes !== true) {
+  if (parsedArgs.yes !== true || scripts.length > 0) {
```

Comment:
```suggestion
  if (parsedArgs.yes !== true && scripts.length > 0) {
```

If they said `yes` AND if they have scripts.. we do NOT want to prompt them.
If they did not say `yes` AND they have scripts... we DO want to prompt.
If they did not say `yes` AND they DO NOT have scripts.... we DO NOT want to prompt.